adapters/repos/db/inverted/bm25_searcher.go

```diff
@@ -183,7 +183,7 @@ func (b *BM25Searcher) wand(
 	indices := make([]map[uint64]int, lengthAllResults)
 
 	var eg errgroup.Group
-	eg.SetLimit(_NUMCPU)
+	eg.SetLimit(4)
```

Comment:
what is the motivation for change?
is `_NUMCPU` too much? should be fixed to 4 or in range <`_NUMCPU`, 4>?